test/core/transport/binder/end2end/fake_binder.cc

```diff
@@ -39,34 +41,41 @@ absl::Status FakeWritableParcel::SetDataPosition(int32_t pos) {
 absl::Status FakeWritableParcel::WriteInt32(int32_t data) {
   data_[data_position_] = data;
   SetDataPosition(data_position_ + 1).IgnoreError();
+  data_size_ += 4;
```

Comment:
Probably use `size_of(int32_t)` instead of hardcoding the value? Same for other types

Reply:
Done.